topic-operator/src/main/java/io/strimzi/operator/topic/K8sTopicWatcher.java

```diff
@@ -44,23 +45,24 @@ public void eventReceived(Action action, KafkaTopic kafkaTopic) {
             }
             LOGGER.info("{}: event {} on resource {} generation={}, labels={}", logContext, action, name,
                     metadata.getGeneration(), labels);
-            Handler<AsyncResult<Void>> resultHandler = ar -> {
-                if (ar.succeeded()) {
-                    LOGGER.info("{}: Success processing event {} on resource {} with labels {}", logContext, action, name, labels);
-                } else {
-                    String message;
-                    if (ar.cause() instanceof InvalidTopicException) {
-                        message = kind + " " + name + " has an invalid spec section: " + ar.cause().getMessage();
-                        LOGGER.error("{}", message);
-
+            if (!action.equals(Action.ERROR)
```

Comment:
why the need for this check on ERROR action?

Reply:
It's essentially the same line of code as was on the old line 63. TBH I've never really known under what circumstances we might see ERROR, but if/when we saw it, it wouldn't mean we should reconcile.